app/services/proofing/resolution/progressive_proofer.rb

```diff
@@ -136,6 +131,11 @@ def should_proof_state_id_with_aamva?
         end
       end
 
+      def aamva_supports_state_id_jurisdiction?
+        state_id_jurisdiction = applicant_pii['state_id_jurisdiction']
+        IdentityConfig.store.aamva_supported_jurisdictions.include?(state_id_jurisdiction)
```

Comment:
WDYT of some stronger assertions here? Technically it's new behavior but it might be nice to know if any bad data slips through loudly + early

```suggestion
        state_id_jurisdiction = applicant_pii.fetch('state_id_jurisdiction')
        raise "blank state_id_jurisdiction" if state_id_jurisdiction.blank?
        IdentityConfig.store.aamva_supported_jurisdictions.include?(state_id_jurisdiction)
```

Reply:
Yeah, this is a bit of a change. Let me do it; based on what I've seen in the tests I'll bet a bunch of unit tests will start failing and need to be tidied.

Reply:
I am going to put this back the way it was. It looks like there is some behavior that depends on this fully functioning when PII is missing:

https://github.com/18F/identity-idp/blob/ec59a21a80b685ec50a4035d59e67b3f9b386486/spec/services/proofing/resolution/progressive_proofer_spec.rb#L171-L185

I am planning on coming through here and modifying the arguments to match the PII structs we added a while back. I'm hoping to come up with a better solution for handling malformed PII at that point.

Reply:
thanks for trying!